build_msvc/msvc-autogen.py

```diff
@@ -58,7 +77,8 @@ def main():
             with open(vcxproj_filename, 'w', encoding='utf-8') as vcxproj_file:
                 vcxproj_file.write(vcxproj_in_file.read().replace(
                     '@SOURCE_FILES@\n', content))
-
+    os.system('copy build_msvc\\bitcoin_config.h src\\config\\bitcoin-config.h')
+    os.system('copy build_msvc\\libsecp256k1_config.h src\\secp256k1\\src\\libsecp256k1-config.h')
```

Comment:
I would prefer to use shutil to copy the files. Also, use absolute file path instead.